pkg/core/resources/apis/meshservice/generate/generator_test.go

```diff
@@ -266,20 +266,25 @@ var _ = Describe("MeshService generator", func() {
 		dp := core_mesh.NewDataplaneResource()
 		Expect(resManager.Delete(context.Background(), dp, store.DeleteByKey("dp-1", model.DefaultMesh))).To(Succeed())
 
+		gracePeriodStartedAt := ""
 		// Wait until the MeshService has been marked with grace period start
 		Eventually(func(g Gomega) {
 			g.Expect(resManager.Get(context.Background(), ms, store.GetByKey("backend", model.DefaultMesh))).To(Succeed())
-			g.Expect(ms.GetMeta().GetLabels()).To(HaveKey("kuma.io/deletion-grace-period-started-at"))
+			g.Expect(ms.GetMeta().GetLabels()).To(HaveKey(mesh_proto.DeletionGracePeriodStartedLabel))
+			gracePeriodStartedAt = ms.GetMeta().GetLabels()[mesh_proto.DeletionGracePeriodStartedLabel]
 		}, "2s", "100ms").Should(Succeed())
+
 		labelExistedSince := time.Now()
```

Comment:
It's confusing that we have both gracePeriodStartedAt and labelExistedSince

Reply:
Let me rename them.